CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 4.12.0
+
+_July 6, 2021_
+
+Big thanks to the 10 contributors who made this release possible. This will be the last v4 release. It includes deprecations that should help develpers to have an easier adoption of v5.
```

Comment:
I doubt it will be the last, we might add more deprecations as we see opportunities, to ease the migration.

```suggestion
Big thanks to the 10 contributors who made this release possible. It includes deprecations that should help developers to have an easier adoption of v5.
```